src/app/query/bitcoin/ordinals/ordinals-aware-utxo.query.ts

```diff
@@ -36,7 +36,7 @@ export async function getNumberOfInscriptionOnUtxo(id: string, index: number) {
 
 async function fetchOrdinalsAwareUtxo(
   txid: string,
-  index: number
+  index?: number
```

Comment:
The type from btc-signer lib has the index as possible undefined so it has to be handled somewhere. Doesn't work to handle it in the `useQueries`.

Reply:
Can we handle elsewhere rather than changing the schema of this fn? Maybe a function: `getIndexWithDefaultZero`?

Reply:
Why would we force the index to be zero? Wouldn't that be problematic to assume?

Reply:
I think you would want the query to fail if the index is undefined? Not force a default?

Reply:
I see this as a leaky abstraction. Because of btc-signer specific implementation problem, other parts of the code are updated to reflect said issue.

But a utxo has both a txid and an index. There's no reason to ever call it without one. So a better solution is to prevent it being called in the first place, rather than update the function to allow illegal cases.

Reply:
Yeah, I agree, I'll figure out a way to catch it earlier.